xbmc/cores/VideoPlayer/DVDFileInfo.cpp

```diff
@@ -267,8 +267,8 @@ bool CDVDFileInfo::CanExtract(const CFileItem& fileItem)
     return false;
 
   // mostly can't extract from discs and files from discs.
-  if (URIUtils::IsBlurayPath(fileItem.GetPath()) || VIDEO::IsBDFile(fileItem) || fileItem.IsDVD() ||
-      fileItem.IsDiscImage() || VIDEO::IsDVDFile(fileItem, false, true))
+  if (fileItem.IsBluray() || fileItem.IsDVD() || fileItem.IsDiscImage() ||
+      VIDEO::IsDVDFile(fileItem, false, true))
     return false;
```

Comment:
What is the benefit? item.IsBluray() is relatively more expensive because indirectly uses GetOpticalMediaPath() which uses disk i/o.
CanExtract is used by the library scanner and can call this function many times. Small differences add up.

Reply:
Also, we're trying to avoid the fileitempalooza with it doing everything, so compartmentalized utilities are preferred (especially ones that don't hit the disk)